test/cmd/certs.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
+
+os::test::junit::declare_suite_start "cmd/admin/certs-validation"
+
+CERT_DIR="${BASETMPDIR:-/tmp}/certs"
+mkdir -p "${CERT_DIR}"
```

Comment:
This looks like the right place to put it, and no, you don't need to worry about cleaning it up.